test/bartlett.jl

```diff
@@ -1,8 +1,9 @@
 using HypothesisTests
 using Test
-using StatsBase
 using DelimitedFiles
 
+using StatsAPI: dof, nobs
```

Comment:
This change isn't actually necessary since StatsBase reexports those

Reply:
I guess I changed it while simplifying other imports and uses of StatsBase since no functionality or definition of StatsBase is needed here.